src/PincodeInput.svelte

```diff
@@ -49,6 +49,7 @@
   }}"
   on:blur
   on:keydown
+  on:input
```

Comment:
This forwarded event is not working when I test it locally like so:

```svelte
<Pincode>
  <PincodeInput on:input={e => console.log('input', e)}  />
  <PincodeInput />
  <PincodeInput />
  <PincodeInput />
</Pincode>
```

Reply:
This is not necessary, just a relict of trying another approach. I'll remove it.

Reply:
>     * You'll also need to apply any changes to the unstyled components located in `src/unstyled`

Why did you decide to separate your component into an unstyled/styled version? This seems redundant and unnecessary, since one could always style the component by using :global selectors. Or I could simplify the design by implementing themes with a css selector?

Reply:
By default, components should have sensible, minimal styles.

Unstyled components give the consumer full control, without the extra generated JS code from scoped styles.